mindbender/maya/tests/test_workflow.py

```diff
@@ -48,6 +48,9 @@ def setup():
     # Setup environment
     os.environ["MINDBENDER_ASSETPATH"] = assetdir
     os.environ["MINDBENDER_SILO"] = "assets"
+    os.environ["MINDBENDER_FPS"] = "24"
```

Comment:
It's great that it passes the tests now, but we really shouldn't make these variables a requirement for the rest of the pipeline.

Think of a way to make these optional. To use and validate when they exist, but do no harm if not. They are an added bonus to the pipeline.